public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -279,6 +283,15 @@ export function PrometheusDatasource(instanceSettings, $q, backendSrv, templateS
     return metricName + '{' + labelPart + '}';
   };
 
+  this.alignTime = function(time, range) {
```

Comment:
This alignment still does not make sense. The alignment should not cause more than a 1-3 percent change in time window. If I have last 5 minutes, and have metrics every second auto refresh to every 5sec I want live, not new data every minute